src/numbers.js

```diff
@@ -1,45 +1,47 @@
-function add (a, b) {
-  // your code here
-}
+
+  const add = (a, b) => {
+    return a + b;
+  };
+
 
 function subtract (a, b) {
-  // your code here
+  return a-b;
```

Comment:
This is inconsistent with the add function above, convention is to have a whitespace either side of the operator
```suggestion
  return a - b;
```